Blueprint.Api.Data/Models/CatalogInject.cs

```diff
@@ -28,6 +28,8 @@ public CatalogInjectEntity(Guid injectId, Guid catalogId)
 
         public Guid CatalogId { get; set; }
         public CatalogEntity Catalog { get; set; }
+        public bool IsNew { get; set; }
```

Comment:
How does something become not new? It seems like this means its been added to a MSEL, but I don't see how it gets updated.

Reply:
"New" is user data and means that it is a new catalog item.  The creator of the catalog decides which injects are "new" and which ones are not.  It is edited on the catalog edit dialog in the UI.